test/resources/examples/index.js

```diff
@@ -91,6 +91,153 @@ const basicRingTrade = generateTestCase({
   ],
 })
 
+const longRingTrade = generateTestCase({
+  name: "Long Ring",
+  orders: [
+    { sellToken: 0, buyToken: 1, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 1, buyToken: 2, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 2, buyToken: 3, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 3, buyToken: 4, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 4, buyToken: 5, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 5, buyToken: 6, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 6, buyToken: 7, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 7, buyToken: 8, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 8, buyToken: 9, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 9, buyToken: 10, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 10, buyToken: 11, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 11, buyToken: 12, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 12, buyToken: 13, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 13, buyToken: 14, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 14, buyToken: 15, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 15, buyToken: 16, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 16, buyToken: 17, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 17, buyToken: 18, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 18, buyToken: 19, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 19, buyToken: 20, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 20, buyToken: 21, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 21, buyToken: 22, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 22, buyToken: 23, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+    { sellToken: 23, buyToken: 24, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
+    { sellToken: 24, buyToken: 0, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
+  ],
+  solutions: [
+    {
+      name: "Long Ring Around the Rosie",
```

Comment:
This name

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2F3oz8xYA4aTzZbxScp2%2Fgiphy.gif&token=rosie" data-gitmeme-token="rosie"><img src="https://media0.giphy.com/media/3oz8xYA4aTzZbxScp2/giphy.gif" title="Created by gitme.me with /rosie"/></a>

Reply:
Will rename.